.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+<!-- Thank you for opening a new pull request. -->
```

Comment:
I'm not sure whether comment in PR template works or not...: https://github.com/apache/arrow/issues/45485

Reply:
Ah, thanks for pointing that out.

I wish GitHub let you build pull request template forms the same way that you can do so for issues...I will update this